python/setup.py

```diff
@@ -298,6 +298,7 @@ def get_outputs(self):
     use_scm_version = {"root": "..", "relative_to": __file__},
     setup_requires=['setuptools_scm'],
     install_requires=['cython >= 0.23', 'numpy >= 1.9', 'six >= 1.0.0'],
+    test_requires=['pytest'],
```

Comment:
We used to have `pandas>=0.12.0` in requirements.txt - is that the version we should go with? Also I'm assuming it should go in `install_requires` yes?